add unstable_enable_feature_access_simulation variable (#130)
common-fate/terraform-aws-common-fate-deployment: .changeset/calm-drinks-melt.md

---

> "@common-fate/terraform-aws-common-fate-deployment": patch

Reviewer: ```suggestion
"@common-fate/terraform-aws-common-fate-deployment": minor
```

---

> Add unstable_enable_feature_access_simulation variable to enable the feature for access simulation apis

Reviewer: ```suggestion
Adds a `unstable_enable_feature_access_simulation` variable to enable Access Simulation (currently in beta) 
```
